### PR TITLE
Add sales number input field to sample app to allow for proper of Dia…

### PR DESCRIPTION
**Pull request by joliveros**

### CODE REVIEW

**Comment by joliveros** [comment]
Here we go Sir @acamino

File: public/app.js [hunk 1]
```diff
@@ -19,7 +19,8 @@ $(function() {
             method: 'POST',
             dataType: 'json',
             data: {
-                phoneNumber: $('#phoneNumber').val()
+                phoneNumber: $('#phoneNumber').val(),
+                salesNumber: $('#salesNumber').val()
```
Comment (juancarlospaco) [inline public/app.js]: What do you think about change:
```javascript
$.ajax({
        url: '/call',
        method: 'POST',
        dataType: 'json',
        data: {
            phoneNumber: $('#phoneNumber').val(),
            salesNumber: $('#salesNumber').val()
        }
}
```
Into:
```javascript
$.post(
        url: '/call',
        dataType: 'json',
        success: null,
        data: {
            phoneNumber: $.trim($('#phoneNumber').val()),
            salesNumber: $.trim($('#salesNumber').val())
        }
)
```
As in:
https://api.jquery.com/jQuery.post/#jQuery-post-url-data-success-dataType
https://api.jquery.com/jQuery.trim/#jQuery-trim-str

File: routes/index.js [hunk 2]
```diff
@@ -56,12 +57,18 @@ module.exports = function(app) {
...
-};
+    app.post('/outbound/:salesNumber', function(request, response) {
+        var salesNumber = request.params.salesNumber;
+        var twimlResponse = new twilio.TwimlResponse();
 
+        twimlResponse.say('Thanks for contacting our sales department. Our ' +
+                          'next available representative will take your call. '
+                          , {
+                            voice: 'alice',
+                          });
```
Comment (juancarlospaco) [inline routes/index.js]: Maybe this one is too short to go splitted apart on 3 lines :grey_question: 
```javascript
{voice: 'alice'}
```
Looks easy and good, not too long for a single line.

File: routes/index.js [hunk 3]
```diff
@@ -35,8 +35,9 @@ module.exports = function(app) {
         // This should be the publicly accessible URL for your application
         // Here, we just use the host for the application making the request,
         // but you can hard code it or use something different if need be
-        var url = 'http://' + request.headers.host + '/outbound';
-        
+        var salesNumber = request.body.salesNumber;
+        var url = 'http://' + request.headers.host + '/outbound/' + encodeURIComponent(salesNumber)
```
Comment (juancarlospaco) [inline routes/index.js]: Missing a semicolon `;` at the end of the line.

**Comment by acamino** [comment]
@juancarlospaco thanks for your review, appreciate it. Yesterday we had a meeting with @joliveros where we defined the scope of this change. We don't want to refactor the existing code. The purpose of this work is to address the requested changes described in https://www.pivotaltracker.com/story/show/135974741.

@joliveros could you please make sure the test suite is passing in Travis - if I'm not wrong it would be just adding the [.travis.yml](https://github.com/TwilioDevEd/clicktocall-node/pull/8/files#diff-354f30a63fb0907d4ad57269548329e3) file.

File: README.md [hunk 4]
```diff
@@ -5,6 +5,8 @@ step-by-step tutorial, [visit this link](https://twilio.com/docs/howto/click-to-
 
 [Read the full tutorial here](https://www.twilio.com/docs/tutorials/walkthrough/click-to-call/node/express)!
 
+[![Build Status](https://travis-ci.org/TwilioDevEd/clicktocall-node.svg?branch=master)](https://travis-ci.org/TwilioDevEd/clicktocall-node)
```
Comment (acamino) [inline README.md]: +1

File: public/app.js (hunk 1 above)
Comment (joliveros) [inline public/app.js]: what do you think about this @acamino? I did notice the numbers come with a bunch of spaces, sometimes dashes. Do you think it's a good idea to clean them up before using them with twilio api? It works as is but don't know if this is bad practice.

**Comment by acamino** [comment]
Thanks @joliveros - LGTM!

**Pull request merged**